src/client/components/ConsentsSubHeader.tsx

```diff
@@ -28,6 +26,7 @@ type Props = {
 	errorMessage?: string;
 	errorContext?: React.ReactNode;
 	isNativeApp?: IsNativeApp;
+	isInAbSimplifyRegFlowTest: boolean;
```

Comment:
```suggestion
	isInAbSimplifyRegFlowTest?: boolean;
```

this could become optional, and you set it to default `false`